specs/commandline/verify.md

```diff
@@ -197,3 +197,39 @@ export NOTATION_EXPERIMENTAL=1
 # The value of --scope should be set base on the trust policy configuration
 notation verify --oci-layout --scope "local/hello-world" hello-world:v1
 ```
+
+### Verify signatures on an OCI artifact with json output
+
+Use the `--output` flag to format verification output in json.
+
+```shell
+notation verify --output json localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
```

Comment:
Just curious, what error message are we expecting from the following command:
`notation verify --output <something_not_defined> localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9`